src/99-appendices/09-entities.md

```diff
@@ -133,13 +136,26 @@ Full name: Echo
 
 Format: `echo-<index>`
 
-Definition: Multi-echo data MUST be split into one file per echo.
-Each file shares the same name with the exception of the `_echo-<index>`
-key/value.
+Definition: If constituents of an entity-linked file collection differ as a
+function of `EchoTime` acquisition parameter, the `_echo-<index>` key/value
+pair MUST be used to distinguish individual files.
```

Comment:
I think this revised definition is worded a little confusingly. Rather than "constituents of an entity-linked file collection", I would say "files that are part of an entity-linked file collection". Instead of "differ as a function of `EchoTime`", I think you could say "have different `EchoTime`s.

Reply:
Also, I think the definition should go before the usage and warning. Something like "The `_echo-<index>` key/value pair represents the `EchoTime` metadata field." Then you can go into when and how it should be used.

Reply:
Okay I just realized I commented on the rendered file instead of the source one 😞. Resolving this and commenting in the right place.